jnim/private/jni_export.nim

```diff
@@ -456,6 +464,22 @@ macro jexport*(a: varargs[untyped]): untyped =
     proc jniRegisterNativeMethods(t: type[`classNameIdent`], `clazzIdent`: JVMClass) =
       `nativeMethodsRegistration`
 
+  block: # Initializer thunk
+    let iClazz = ident"clazz"
+    let classIdent = ident(className)
+    let thunkName = ident(JniExportedFunctionPrefix & className & "_" & InitializerName.replace("_", "_1"))
```

Comment:
I dont't understand what this `replace` does.

Reply:
https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/design.html#wp133 "We adopted a simple name-mangling scheme to ensure that all Unicode characters translate into valid C function names. (...)"